README.md

```diff
@@ -35,9 +33,73 @@ The package provides one macro, `@match`, which can be used as:
         _                     => default_result
     end
 
-It is possible to supply variables inside pattern, which will be bound
-to corresponding values. 
+    if @ismatch value pattern
+        # Code that uses variables bound in the pattern
+    end
 
 See the [documentation](https://JuliaServices.github.io/Match.jl/stable/)
```

Comment:
this link 404's should it work?

Reply:
After we do a release with the doc automation it should get generated.